docs/source/development-environment.rst

```diff
@@ -0,0 +1,141 @@
+Zipline Development Guidelines
+==============================
+
+Environment Setup
+-----------------
+
+First, you'll need to clone zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:quantopian/zipline.git
+
+Then check out to a new branch where you can make your changes:
+
+.. code-block:: bash
+		
+   git checkout -b some-short-descriptive-name
+
+The following section assumes you already have virtualenvwrapper and pip installed on your system. If you don't already have them, you'll need some C library dependencies. On Linux you can run:
+
+.. code-block:: bash
+
+   # on linux
+   $ sudo apt-get install libopenblas-dev liblapack-dev gfortran
+
+   
+   $ wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
+   $ tar -xvzf ta-lib-0.4.0-src.tar.gz
+   $ cd ta-lib/
+   $ ./configure --prefix=/usr
+   $ make
+   $ sudo make install
+
+And for `ta-lib` on OS X you can just run: `brew install ta-lib`.
+
+Suggested installation of Python library dependencies used for development:
+
+.. code-block:: bash
+
+   $ mkvirtualenv zipline
+   $ ./etc/ordered_pip.sh ./etc/requirements.txt
+   $ pip install -r ./etc/requirements_dev.txt
+   $ pip install -r ./etc/requirements_blaze.txt 
+   $ pip install -r ./etc/requirements_talib.txt 
+   $ pip install coverage coveralls
+
+Finally, you can install zipline in development mode by running:
+
+.. code-block:: bash
+
+   python setup.py built_ext --inplace
+
+
+Style Guide & Running Tests
+---------------------------
+
+We use `flake8` for checking style requirements and `nosetests` to run zipline tests.
+
+Before submitting patches or pull requests, please ensure that your changes pass when running:
+
+.. code-block:: bash
+
+   $ flake8 zipline
+
+and
+
+.. code-block:: bash
+		
+   $ nosetests --with-coverage
+
+
+If you get an error running nosetests after setting up a fresh virtualenv, please try running deactivate zipline; workon zipline, where zipline is the name of your virtualenv.
+
+   
+Docs
+----
+
+To build and view the docs locally, run:
+
+.. code-block:: bash
+
+   $ cd docs
+   $ make html
+   $ {BROWSER} build/html/index.html
+
+
+Commit messages
+---------------
+
+Standard acronyms to start the commit message with are:
```

Comment:
Feels a bit stilted to have the preposition before the verb at the end there. How about "Standard prefixes to start a commit message"?